File: Frontend/erp/src/app/core/models/filter.model.ts
import { PagingModel } from './paging.model';

export class FilterModel{
  text:string;
  paging: PagingModel;
  candidateId : string;
  jobId : number;
  processId: number;
  constructor(){
    this.text = '';
    this.paging = new PagingModel();
    this.candidateId = '';
    this.processId = 0;
    this.jobId = 0;
  }
}

File: Backend/Core/Core.CommonModel/FilterModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Core.CommonModel
{
    public class FilterModel
    {
        public string Text { get; set; }
        public string CandidateId { get; set; }
        public int ProcessId { get; set; } = 0;
        public int JobId { get; set; } = 0;
        public PagingModel Paging { get; set; } = new PagingModel();
    }
}
